.tekton/ci-helper-app-pull-request.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: ci-helper-app-pull-request
+  annotations:
+    pipelinesascode.tekton.dev/on-event: "[pull_request]"
+    pipelinesascode.tekton.dev/on-target-branch: "[main]"
+    pipelinesascode.tekton.dev/max-keep-runs: "5"
```

Comment:
under which namespace would this PLR be running?

Reply:
that depends on the namespace where [related repository CR is created](https://github.com/redhat-appstudio/infra-deployments/blob/main/components/konflux-ci/base/repository.yaml). in this case it's `konflux-ci`